nrfutil/device/logLibVersions.ts

```diff
@@ -101,15 +103,15 @@ export default async (moduleVersion: ModuleVersion) => {
                     JLinkArchOnDarwin === 'arm'
                         ? '64-bit Apple M1 Installer'
                         : '64-bit Intel Installer';
-                logger.warn(
+                logger?.warn(
                     `It looks like you have installed JLink using ${JLinkInstallerVersion}, but currently we only support their Universal Installer for your system.`
                 );
-                logger.warn(
+                logger?.warn(
                     `Please install JLink: https://www.segger.com/downloads/jlink/JLink_MacOSX_V788j_universal.pkg`
                 );
             }
         }
     } catch (error) {
-        logger.logError('Failed to get the library versions', error);
+        logger?.error('Failed to get the library versions', error);
```

Comment:
Please check whether this is correct, I assume the change from `logError` to `error` is a mistake: [logError](https://github.com/NordicSemiconductor/pc-nrfconnect-shared/blob/424faf72d87bd96ebb95fce0aba2132587d30f7e/src/logging/index.ts#L96-L101) is a function which outputs the second argument. With `error` the second argument is considered [“meta”](https://github.com/winstonjs/winston/tree/master#streams-objectmode-and-info-objects) and probably discarded.

Reply:
You're correct.
I had not found `logError` (though I obviously didn't check very thoroughly) and this gave me type errors. I'll see whether I will just string format this or fix the types.

Reply:
Decided to reimplement the logError explicitly here.
The original logError was part of the `SharedLogger` which I did not want to reference here as we should only rely on the `nrfutilLogger`.